docs/getting-started/example-bpf-local.md

```diff
@@ -1,126 +1,192 @@
 # Deploying Example eBPF Programs On Local Host
 
 This section describes running bpfman and the example eBPF programs on a local host.
-When running bpfman, it can be run as a process or run as a systemd service.
-Examples run the same, independent of how bpfman is deployed.
-
-### Building
 
-To build directly on a system, make sure all the prerequisites are met, then build.
+## Example Walk Through
 
-#### Prerequisites
+Assume the following command is run:
 
-_This assumes bpfman is already installed and running on the system.
-If not, see [Setup and Building bpfman](./building-bpfman.md)._
+```console
+cd bpfman/examples/go-xdp-counter/
+sudo ./go-xdp-counter -iface eno3
+```
 
-1. All [requirements defined by the `cilium/ebpf` package](https://github.com/cilium/ebpf#requirements)
-2. libbpf development package to get the required eBPF c headers
+The diagram below shows `go-xdp-counter` example, but the other examples operate in
+a similar fashion.
 
-   Fedora:
+![go-xdp-counter On Host](../img/gocounter-on-host.png)
 
-   `sudo dnf install libbpf-devel`
+Following the diagram (Purple numbers):
 
-   Ubuntu:
+1. When `go-xdp-counter` userspace is started, it will send a gRPC request over unix
+   socket to `bpfman-rpc` requesting `bpfman` to load the `go-xdp-counter` eBPF bytecode located
+   on disk at `bpfman/examples/go-xdp-counter/bpf_bpfel.o` at a priority of 50 and on interface `eno3`.
+   These values are configurable as we will see later, but for now we will use the defaults
+   (except interface, which is required to be entered).
+2. `bpfman` will load it's `dispatcher` eBPF program, which links to the `go-xdp-counter` eBPF program
+   and return a kernel Program ID referencing the running program.
+3. `bpfman list` can be used to show that the eBPF program was loaded.
+4. Once the `go-xdp-counter` eBPF bytecode is loaded, the eBPF program will write packet counts
+   and byte counts to a shared map.
+5. `go-xdp-counter` userspace program periodically reads counters from the shared map and logs
+   the value.
 
-   `sudo apt-get install libbpf-dev`
+Below are the steps to launch `bpfman`, run some example programs, and use the `bpfman` CLI to load
+and unload other eBPF programs.
 
-3. Cilium's `bpf2go` binary
+## Launching bpfman
```

Comment:
@astoycos The removal of `3. Cilium's `bpf2go` binary` is throwing me off a little, because that line is in a totally different file. I think you are asking to move the existing `Launching bpfman` section into it's own file, right?

Reply:
Yeah! Idk why the diff looked like that sorry

Reply:
That's what I did in the last commit. Let me know if you wanted otherwise.